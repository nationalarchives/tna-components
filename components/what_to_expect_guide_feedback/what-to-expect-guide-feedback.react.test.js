import React from 'react';
import WTEGFeedbackWidget from './what-to-expect-guide-feedback.react';
import TestRenderer from 'react-test-renderer';

// // Snapshot 1
it('renders correctly', () => {
  const tree = TestRenderer.create(<WTEGFeedbackWidget />).toJSON();
  expect(tree).toMatchSnapshot();
});
