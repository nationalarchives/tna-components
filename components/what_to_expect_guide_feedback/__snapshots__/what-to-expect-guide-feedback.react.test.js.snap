// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <fieldset
      className="initial-question"
      id={null}
    >
      <legend>
        Did you find this guide helpful?
      </legend>
      <button
        onClick={[Function]}
        type="button"
      >
        Yes
      </button>
      <button
        onClick={[Function]}
        type="button"
      >
        No
      </button>
    </fieldset>
  </form>
</div>
`;
